README.md

```diff
@@ -314,6 +315,8 @@ Install dependencies [(See below)](#dependencies-1)
 ```sh
 git clone "https://github.com/pystardust/ani-cli.git"
 sudo cp ani-cli/ani-cli /usr/local/bin
+mkdir "${HOME}/.config/ani-cli"
+cp -rfv ani-cli/scripts "${HOME}/.config/ani-cli"
```

Comment:
Definitely don't store the script in `~/.config/ani-cli`, a better candidate would be `$XDG_DATA_HOME/ani-cli` or `~/.local/share/ani-cli`.